virtcontainers/pkg/cgroups/manager.go

```diff
@@ -331,3 +331,24 @@ func (m *Manager) RemoveDevice(device string) error {
 	m.Unlock()
 	return fmt.Errorf("device %v not found in the cgroup", device)
 }
+
+func (m *Manager) UpdateCpuSets(cpuset string) error {
+	cgroups, err := m.GetCgroups()
+	if err != nil {
+		return fmt.Errorf("could not get original cgroups: err: %v", err)
+	}
+
+	m.logger().WithField("map-length", len(m.GetPaths())).Debug("EGERNST")
```

Comment:
ummmm...? :smile:

Reply:
Sorry, I have a 'debug' commit on the top of this, as there are some issues in the interactions with virtcontainers/pkg/cgroups.  journalctl -f | grep EGERNST :)